test_assets/beacon/deneb/SignedBeaconBlock/ssz_random/case_0/value.yaml

```diff
@@ -91,8 +91,8 @@ message:
         signature: '0xfb09452cee022c73ae85b69c45c7cb45a20c369eeb985c31183124acb770eaf77728708286c6e68b9da0a63b197843b31c9d40ea3637a39915dee862b75922a4c72d2647cea8441e68cb7b0dca52eff1c84b38cfc8a8cabf8250e34334ed34b4'
       - aggregation_bits: '0x6806'
         data:
-          slot: 4535242405270891733
-          index: 9093395152080509436
+          slot: "4535242405270891733"
+          index: "9093395152080509436"
```

Comment:
Those should be updated in other places below. Also, we need to do the same with the `case_1/value.yaml` file.

Reply:
I don't see why it should be updated everywhere

Reply:
If the attestation format is well-specified as always being a string, then we should have all our tests match the spec, and we don't need to cover the int case. If it's not well-specified and it comes in different types from different sources, then it makes sense to have a string and int version of every test case.

What's our go-to source for these specs? I see:
- https://www.quicknode.com/docs/ethereum/eth-v1-beacon-blocks-block_id-attestations which says string, but it's a 3rd-party source instead of a primary source
- https://github.com/ethereum/consensus-specs/blob/1b408e9354358cd7f883c170813e8bf93c922a94/specs/phase0/beacon-chain.md#attestationdata that seems to suggest an int value, but this isn't necessarily the same as the RPC spec

Reply:
> If the attestation format is well-specified as always being a string, then we should have all our tests match the spec, and we don't need to cover the int case. If it's not well-specified and it comes in different types from different sources, then it makes sense to have a string and int version of every test case.
> 
> What's our go-to source for these specs? I see:
> 
> * https://www.quicknode.com/docs/ethereum/eth-v1-beacon-blocks-block_id-attestations which says string, but it's a 3rd-party source instead of a primary source
> * https://github.com/ethereum/consensus-specs/blob/1b408e9354358cd7f883c170813e8bf93c922a94/specs/phase0/beacon-chain.md#attestationdata that seems to suggest an int value, but this isn't necessarily the same as the RPC spec

The yaml format in `consensus-spec-tests` uses int's. Those yaml files are taken from `consensus-spec-tests`.

The beacon-api uses strings as JSON doesn't support u64.

So I don't see why we would deviate from `consensus-specs-repo` in how they define their format.

> then it makes sense to have a string and int version of every test case.

Then I will just re-add my json tests back, instead of modifying the yaml files

Reply:
All the beacon yaml files use numbers instead of strings and we have over 10 yaml files at least from `consensus-spec-tests`